15/umbraco-cms/reference/management-api/external-access.md

```diff
@@ -0,0 +1,95 @@
+---
+description: How external applications can consume the Management API.
+---
+
+# External access to the Management API
+
+The Management API can be used directly for integrations between Umbraco and external systems.
+
+When consuming the Management API from an external source, you must use the OpenId Connect Client Credentials flow for authorization. Refer to the [API Users](../../fundamentals/data/users/api-users.md) article for details on setting up Client Credentials.
+
+With a set of Client Credentials in place, you can obtain an access token from the Management API token endpoint: `/umbraco/management/api/v1/security/back-office/token`.
+
+The token endpoint response looks like this:
+
+```json
+{
+  "access_token": "ZnEAKg5YwDc7621y6xZlEdT9kwp_ULGQPc5mnY9cDw0",
+  "token_type": "Bearer",
+  "expires_in": 299
+}
+```
+
+As shown, the access token should be used as a Bearer token when consuming the Management API.
+
+Also, notice that access tokens have a fixed expiry. While you can keep issuing new tokens for the Client Credentials, please reuse tokens within their lifespan. This will be more performant and avoid flooding the Umbraco database with tokens.
+
+{% hint style="info" %}
+The Management API does not support OIDC Discovery. This is reserved for accessing protected for Members via the [Delivery API](../content-delivery-api/protected-content-in-the-delivery-api.md).
```

Comment:
A suggestion to avoid using an acronym here, and then I believe the word "content" was missing 🙈 

```suggestion
The Management API does not support OpenID Connect Discovery. This is reserved for Members accessing protected content via the [Delivery API](../content-delivery-api/protected-content-in-the-delivery-api.md).
```